doc/cephadm/monitoring.rst

```diff
@@ -93,6 +93,33 @@ completed, you should see something like this from ``ceph orch ls``
   node-exporter      2/2  6s ago     docker.io/prom/node-exporter:latest             e5a616e4b9cf  present
   prometheus         1/1  6s ago     docker.io/prom/prometheus:latest                e935122ab143  present
 
+Configuring TLS for Grafana
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
I wonder if it would make sense to add a small paragraph here that explains what this is about?

Reply:
You may want to review that paragraph.

Reply:
```suggestion
``cephadm`` will deploy Grafana using the certificate defined in the ceph key/value store. If a certificate is not specified, ``cephadm`` will generate a self-signed certificate during deployment of the Grafana service.

A custom certificate can be configured using the following commands.
```

nit - this paragraph is a little verbose, maybe it can be condensed?

Reply:
It is indeed a little verbose. Thanks for your suggestion. I'll use it.